cmd/urunc/create.go

```diff
@@ -68,16 +68,26 @@ var createCommand = cli.Command{
 		if err := checkArgs(context, 1, exactArgs); err != nil {
 			return err
 		}
-		err := handleNonBimaContainer(context)
-		if err != nil {
-			return err
-		}
 
 		if !context.Bool("reexec") {
 			containerID := context.Args().First()
 			metrics.Capture(containerID, "TS00")
+			err := handleQueueProxy(context)
+			if err != nil {
+				return err
+			}
+			err = handleNonBimaContainer(context)
```

Comment:
Given the last changes, it seems we do not need this anymore, since we will never each this point.

Reply:
Sure, i removed the redundant code.